packages/backend/src/analytics/LightdashAnalytics.ts

```diff
@@ -452,6 +455,22 @@ type AnalyticsDashboardView = BaseTrack & {
     };
 };
 
+type PinnedListUpdated = BaseTrack & {
+    event: 'pinned_list.updated';
+    userId: string;
+    properties: {
+        projectId: string;
+        organizationId: string;
+        source: 'homepage' | Space['name'];
```

Comment:
I don't think `source` is adequate for this. Seems like it was from where the update was made. `location` seems more appropriate.  And we can have `homepage` and the only value for now and later on have `space`.

```suggestion
        location: 'homepage';
```